src/ResourceManager/IotCentral/Commands.IotCentral/help/Get-AzureRmIotCentralApp.md

```diff
@@ -0,0 +1,237 @@
+---
+external help file: Microsoft.Azure.Commands.IotCentral.dll-Help.xml
+Module Name: AzureRM.IotCentral
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.iotcentral/get-azurermiotcentralapp
+schema: 2.0.0
+---
+
+# Get-AzureRmIotCentralApp
+
+## SYNOPSIS
+Gets properties for either one or several IoT Central Applications.
+
+## SYNTAX
+
+### ListIotCentralAppsParameterSet (Default)
+```
+Get-AzureRmIotCentralApp [[-ResourceGroupName] <String>] [-AsJob] [-DefaultProfile <IAzureContextContainer>]
```

Comment:
Is there a reason we have 2 brackets here `[[-ResourceGroupName] <String>]`. It doesn't look like the following command has this.

Reply:
this means the parameter is positional (both the switch and the parameter value are optional

Reply:
This is standard markdown format.  The links will not work because the docs have not been published yet (they won't until the module is live)

Reply:
Sounds good. Thanks for the info!